ckanext/stcndm/helpers.py

```diff
@@ -237,3 +239,15 @@ def get_by_code(self, code):
             en_text = fr_text = None
 
         return en_text, fr_text
+
+def codeset_choices(codeset_type):
+    """
+    Return a dictionary of {codeset_value: title} for the codeset_type
+    passed
+    """
+    lc = ckanapi.LocalCKAN()
+    r = lc.action.package_search(
+        q='type=codeset',
+        fq='codeset_type=' + codeset_type,
+        rows=1000)
+    return dict((r['codeset_value'], r['title']) for r in r['results'])
```

Comment:
@marcfor this is a pretty simple helper. I'm using the `q` and `fq` parameters but I could have combined this into a single `q` parameter with an `AND`